modules.json

```diff
@@ -214,5 +214,22 @@
         "production": "https://unpkg.com/zone.js@[version]/dist/zone.min.js"
       }
     }
+  },
+  "preact": {
+    "var": "preact",
+    "versions": {
+      ">= 2.4.0": {
+        "development": "https://unpkg.com/preact@[version]/dist/preact.js",
+        "production": "https://unpkg.com/preact@[version]/dist/preact.min.js"
+      }
+    }
+  },
+  "preact-compat": {
+    "versions": {
```

Comment:
`"var": "preactCompat"`

Reply:
What is the use of `var` technically?  Is it the namespace under which it is exported?